modules/mod_ginger_collection/filters/filter_collection_query_offset.erl

```diff
@@ -9,8 +9,10 @@
 
 -include_lib("zotonic.hrl").
 
-collection_query_offset(Current, Pagelen, undefined, _Context) ->
-    get_offset(Current, Pagelen, 0);
+collection_query_offset(Current, Pagelen, undefined, Context) ->
+    collection_query_offset(Current, Pagelen, 1, Context);
+collection_query_offset(Current, undefined, Page, Context) ->
+    collection_query_offset(Current, 15, Page, Context);
```

Comment:
@loetie Is it expected that `Pagelen` can sometimes be undefined? And if so, is `15` a good default?

Reply:
@ddeboer Set this as the default because in mod_ginger_collection on de page.beeldenzoeker_query.tpl this is used as the default pagelen. Not sure if it is actually needed.